src/block/block_write.c

```diff
@@ -39,7 +39,7 @@ __wt_block_truncate(WT_SESSION_IMPL *session, WT_BLOCK *block, wt_off_t len)
      * backups, which only copies log files, or targeted backups, stops all block truncation
```

Comment:
(Not related to this, but at the top of the diff just for a place to put it) Please look through the documentation in `src/docs` and see if there is any need to update discussion about checkpoint deletion and backup cursors anywhere.

Reply:
Good point. I found two places that describe the old behavior and have updated them.